tools/rimraf.js

```diff
@@ -0,0 +1,254 @@
+/*
```

Comment:
we have a tiny change to rimraf and it is unlikely it will be merged to rimraf any time soon (it is unmaintained). Keeping this file in the source tree seems to be not that bad